### ISSUE TITLE
 Unable to load DLL 'node.dll': The specified module could not be found.

### ISSUE THREAD

**Issue by stlsw** [tjanczuk/edge#588]
Dear Author, 

I am trying to use edge.js assembly to start node.js application as a scripting mechanism. but I face issue " Unable to load DLL 'node.dll': The specified module could not be found." when I start running it from C# console application.

what are external libraries:

const uniqid = require('uniqid')
const shortid = require('shortid')
const writeFile = require('write');

I tried to follow the guideline to place all the modules in right place, and in other multiple places for troubleshooting, but it doesn't work.  I am using edge.js 6.11.2. my machine installed node.js is 6.10.0.

Please help

**Comment by wazzamatazz** [comment]
I realise that this issue has been open for ages, but the following may be useful, as I just encountered (and fixed) a similar issue.

If the suggestions in #565 do not resolve your issue, the problem might be that you don't have the [Visual C++ Runtime installed](https://support.microsoft.com/en-gb/help/2977003/the-latest-supported-visual-c-downloads). This is not listed in the requirements for EdgeJS on Windows, but it appears to be a pre-requisite.  I found that I could not run a C# console app that used EdgeJS for scripting on a fresh Windows 2016 VM. Installing the C++ runtime from the link above immediately fixed the problem.

**Comment by LucidObscurity** [comment]
I've also found that the C++ runtime is a requirement or else I receive the same error. I did not need to install node.js locally, just the C++ runtime. The version I installed was 14.22.27821, though these should be backward compatible for the last several years. Before installing the C++ runtime I receive the following:
`Unhandled Exception: System.DllNotFoundException: Unable to load DLL 'node.dll': The specified module could not be found. (Exception from HRESULT: 0x8007007E)
   at EdgeJs.Edge.NodeStartx64(Int32 argc, String[] argv)
   at EdgeJs.Edge.<>c__DisplayClass14_0.<Func>b__0()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()`
